### ISSUE TITLE
DeleteArtifactsNotInTemplate not supported

### ISSUE THREAD

**Issue by jochenschroeer** [Azure/Synapse-workspace-deployment#23]
According to documentation DeleteArtifactsNotInTemplate  should be supported (we would like to have a clean workspace which only contains artifacts from the deployment) but it does not work, see below:

Unexpected input(s) 'DeleteArtifactsNotInTemplate', valid inputs are ['TargetWorkspaceName', 'TemplateFile', 'ParametersFile', 'OverrideArmParameters', 'Environment', 'resourceGroup', 'clientId', 'clientSecret', 'subscriptionId', 'tenantId', 'activeDirectoryEndpointUrl', 'resourceManagerEndpointUrl']

we use the github action as follows (the version is always required which is currently not described in your usage section):

   - name: 'Synapse workspace deployment'
        uses: Azure/Synapse-workspace-deployment@v0.2
        with: 
           ....
             DeleteArtifactsNotInTemplate: true

**Comment by lordozb** [comment]
Looks like the documentation on the market places is being referred from the master branch. 

DeleteArtifactsNotInTemplate is currently not part of the v0.2 release. In order to unblock you can use the master branch. 
Please Replace Azure/Synapse-workspace-deployment@v0.2 by Azure/Synapse-workspace-deployment@master
